cli/templatecreate.go

```diff
@@ -139,7 +139,7 @@ func (r *RootCmd) templateCreate() *serpent.Command {
 				return err
 			}
 
-			if !uploadFlags.stdin() {
+			if !uploadFlags.stdin(inv) {
 				_, err = cliui.Prompt(inv, cliui.PromptOptions{
 					Text:      "Confirm create?",
```

Comment:
We could check if stdin starts with `y` or `yes` since that currently confirms this prompt, but only if we wanted to be really cautious as to not break an already weird usage,